common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_action.yaml

```diff
@@ -265,8 +265,10 @@ eActionOp_APMANAGER:
 
   ACTION_APMANAGER_SUPPORTED_CHANNELS_REQUEST: 72
   ACTION_APMANAGER_SUPPORTED_CHANNELS_RESPONSE: 73
+  ACTION_APMANAGER_READ_ACS_REPORT_REQUEST: 74
+  ACTION_APMANAGER_READ_ACS_REPORT_RESPONSE: 75
```

Comment:
I don't understand why did you add it and create additional states for it on the son_slave, when the ap_manager sends it now on APMANAGER_JOINED_NOTIFICATION?

Reply:
APMANAGER_JOINED_NOTIFICATION will send it only on connect, but ACS_REPORT is used for channel preference query. Still not sure that added logic is correct. Should I use acs report for channel preference or read supported channels?
@tomereli @arnout could you please take a look at this part of code? I added logic to read acs report when getting channel preference. For netlink acs report will just call read_supported_channels, but for dwpal it calls different cmd.

Reply:
State was removed.